code/edgeconsent/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val mavenEdgeVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.edge.consent"
+    enableSpotlessPrettierForJava = true
+    enableCheckStyle = true
+ 
+    publishing {
+        gitRepoName = "aepsdk-edgeconsent-android"
+        addCoreDependency(mavenCoreVersion)
+        addMavenDependency("com.adobe.marketing.mobile", "edge", mavenEdgeVersion)
```

Comment:
nit: 
You can replace it with `addEdgeDependency`
https://github.com/adobe/aepsdk-commons/blob/dev/android/aepsdk-gradle-plugin/aepsdk-gradle-plugin/src/main/kotlin/com/adobe/marketing/mobile/gradle/AEPLibraryExtension.kt#L70